CMakeLists_files.cmake

```diff
@@ -134,7 +134,10 @@ if(OPENCL_FOUND)
   list (APPEND MAIN_SOURCE_FILES opm/simulators/linalg/bda/opencl/openclSolverBackend.cpp)
   list (APPEND MAIN_SOURCE_FILES opm/simulators/linalg/bda/opencl/openclWellContributions.cpp)
 endif()
-if(CUDA_FOUND OR OPENCL_FOUND OR HAVE_FPGA OR HAVE_AMGCL)
+if(HAVE_ROCALUTION)
```

Comment:
this should be ROCALUTION_FOUND

Reply:
5a5107b12
I left the `HAVE_FPGA` variable, because it will be removed in the near future.